pkg/storage/sst.go

```diff
@@ -50,6 +50,7 @@ func CheckSSTConflicts(
 	reader Reader,
 	start, end MVCCKey,
 	leftPeekBound, rightPeekBound roachpb.Key,
+	nowNanos int64,
```

Comment:
I don't think passing in `nowNanos` is necessary here. Every time we make a stats adjustment for a key, we need to age the stats to that key's timestamp, and the returned result here will be processed with `Add()` which automatically adjusts for `LastUpdateNanos` regardless of its current value. If the caller really wants the stats at a particular timestamp (they usually don't), they can simply call `AgeTo()` on it themselves.